src/internal/cluster/common.go

```diff
@@ -28,7 +28,7 @@ var labels = k8s.Labels{
 
 // NewClusterOrDie creates a new cluster instance and waits up to 30 seconds for the cluster to be ready or throws a fatal error.
 func NewClusterOrDie() *Cluster {
-	c, err := NewClusterWithWait(defaultTimeout, true)
+	c, err := NewClusterWithWait(DefaultTimeout, true)
```

Comment:
DefaultTimeout should be replaced in other places that use the 30 second timeout for clusters like line 224 of `deploy.go`.

Reply:
Changes made